src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/SøknadOvergangsstønadMapper.kt

```diff
@@ -34,7 +35,9 @@ class SøknadOvergangsstønadMapper(private val dokumentServiceService: Dokument
 
         OvergangsstønadValidering.validate(søknad)
 
-        return SøknadRequestData(SøknadMedVedlegg(søknad, vedlegg.values.map { it.vedlegg }.flatten()), vedleggData)
+        return SøknadRequestData(SøknadMedVedlegg(søknad,
+                                                  vedlegg.values.map { it.vedlegg }.flatten(),
```

Comment:
Kan man bruke `vedlegg.values.flatMap { it.vedlegg}`?

Reply:
9da6d18